src/nl/documentatie/10.woocommerce-v5.0.md

```diff
@@ -0,0 +1,329 @@
+# WooCommerce
+
+<Stack class="grid-flow-col auto-cols-max gap-1">
+    <GitHubShield repo="myparcelnl/woocommerce/" />
+    <GitHubShield repo="myparcelnl/woocommerce" link="issues" />
+    <GitHubShield repo="myparcelnl/woocommerce" link="pulls" />
+</Stack>
+
+## Introductie
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_plugin_api-key.jpeg" alt="MyParcel plugin API key" />
+
+## Eenmalige handeling
+
+Voor het in gebruik nemen van de WooCommerce MyParcel plugin, moet er eerst een API key worden ingevuld. De API key is gratis te verkrijgen via <a href="https://backoffice.myparcel.nl/registration" target="_blank"> API instellingen in de MyParcel Backoffice</a>.
+Na het invullen van de API key worden alle instellingen van de MyParcel plugin getoond.
+
+<hr/>
+
+## Configuratie
+
+### Algemeen
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_a_nl.jpeg" alt="woocommerce plugin general" />
```

Comment:
Het `alt` attribuut moet een menselijk-leesbare beschrijving zijn van het plaatje, niet gewoon een titel. Dus bijvoorbeeld "Algemene instellingen in de MyParcel WooCommerce plug-in". Geldt ook voor andere `img` tags